pkg/replicate/scheme.go

```diff
@@ -203,9 +204,35 @@ func (rs *replicationScheme) execute(ctx context.Context) error {
 		return availableBlocks[i].BlockMeta.MinTime < availableBlocks[j].BlockMeta.MinTime
 	})
 
+	// Replicate concurrently
```

Comment:
```suggestion
	// Replicate concurrently.
```

https://thanos.io/tip/contributing/coding-style-guide.md/#comments-should-be-full-sentences